src/router/index.js

```diff
@@ -1,14 +1,57 @@
 import { createRouter, createWebHashHistory } from 'vue-router';
 
+const userIsLoggedIn = () => {
+  return localStorage.getItem('UID') !== null;
```

Comment:
there is a firebase method to get current users. Use that for this, not the local storage @ShivanshSinghnnovative